smart-contract/assembly/contracts/internals/chunks.ts

```diff
@@ -22,6 +22,7 @@ export function _setFileChunk(
   index: u32,
   chunk: StaticArray<u8>,
 ): void {
+  // TODO - What if the file is not initialized?
```

Comment:
Should that be fixed i nthis PR or a later one ?

Reply:
No but I'll remove the todo and create an issue if needed